sdk/tables/azure-data-tables/azure/data/tables/_encoder.py

```diff
@@ -78,6 +80,12 @@ def prepare_value(  # pylint: disable=too-many-return-statements
             raise TypeError(f"Unsupported data type '{type(value)}' for entity property '{name}'.")
         raise TypeError(f"Unsupported data type '{type(value)}'.")
 
+    def _is_int32(self, value: int) -> bool:
+        return value >= MIN_INT32 and value <= MAX_INT32
+
+    def _is_int64(self, value: int) -> bool:
+        return not self._is_int32(value) and value >= MIN_INT64 and value <= MAX_INT64
```

Comment:
@annatisch I miss context here, but instinctively, I think that `_is_int64` should return `True` for int32 as well, as if we're scared of overlap we can probably write code like:

```python
if self._is_int32(value):
   # do something int32
elif self._is_int64(value):
   # do something int64, we know it's not int32 already
```

Or at least the function name is misleading on the intent

Reply:
Updated